ui/raidboss/data/07-dt/hunts/shaaloani.ts

```diff
@@ -434,6 +562,233 @@ const triggerSet: TriggerSet<Data> = {
+          // swap in case dir2 is null, and its more natural phrasing
+          [dir2, dir3] = [dir3, dir2];
+          data.ttokSandOrbOnSet++;
+        }
+        if (dir2 && dir3) {
+          return output.triple!({
+            inOut: output[inOut]!(),
+            dir2: output[dir2]!(), // front or back
+            dir3: output[dir3]!(), // right or left
+          });
+        }
+        if (dir2) {
+          return output.double!({ inOut: output[inOut]!(), dir2: output[dir2]!() });
+        }
+        return output[inOut]!();
+      },
+      outputStrings: ttokrroneTempestSandspoutOutputStrings,
+    },
+    // Aoe
+    {
+      id: 'Hunt Ttokrrone Touchdown',
+      type: 'StartsUsing',
+      netRegex: { id: '91DB', source: 'Ttokrrone', capture: false },
+      durationSeconds: 4,
+      response: Responses.aoe(),
+      run: (data) => {
+        // Reset everything as orb sequences start with a touchdown.
+        data.ttokSandOrbSets = 0;
+        data.ttokSandOrbOnSet = 0;
+        data.ttokSandOrbs = [];
+        data.ttokSandOrbPatterns = [];
+        data.ttokRotated = 0;
+      },
+    },
+    // Front cleave, rotates C or CCW and cleaves 4 or 7 times (maybe health dependent?)
+    {
+      id: 'Hunt Ttokrrone Fangward Dustdevil',
+      type: 'StartsUsing',
+      netRegex: { id: ['91C9', '91C5'], source: 'Ttokrrone', capture: false },
+      durationSeconds: 6,
+      suppressSeconds: 1,
+      alertText: (_data, _matches, output) => output.outOfHitbox!(),
+      infoText: (_data, _matches, output) => {
+        return output.rotateFront!();
+      },
+      outputStrings: ttokrroneDustdevilOutputStrings,
+    },
+    // Rear cleave (boss turns to do front cleaves) then rotates C or CCW and cleaves 4 or 7 times (?)
+    {
+      id: 'Hunt Ttokrrone Tailward Dustdevil',
+      type: 'StartsUsing',
+      netRegex: { id: ['91CA', '91C6'], source: 'Ttokrrone', capture: false },
+      durationSeconds: 6,
+      suppressSeconds: 1,
+      alertText: (_data, _matches, output) => output.outOfHitbox!(),
+      infoText: (_data, _matches, output) => {
+        return output.rotateRear!();
+      },
+      outputStrings: ttokrroneDustdevilOutputStrings,
+    },
+    // Dashes around 6 times
+    {
+      id: 'Hunt Ttokrrone Landswallow',
+      type: 'StartsUsing',
+      netRegex: { id: '96F2', source: 'Ttokrrone', capture: false },
+      durationSeconds: 15,
+      infoText: (_data, _matches, output) => output.dodgeToRight!(),
+      outputStrings: {
+        dodgeToRight: {
+          en: 'Go to safe side of first dash => move in after',
+        },
+      },
+    },
+    // The sand spheres cast "Sandburst" to explode. 994E is 2 seconds shorter cast (5.7s)
+    {
+      id: 'Hunt Ttokrrone Sand Spheres Sandburst',
+      type: 'StartsUsing',
+      netRegex: { id: ['994D', '994E'], source: 'Sand Sphere', capture: true },
+      delaySeconds: (_data, matches) => matches.id === '994E' ? 0 : 2,
+      durationSeconds: 5,
```

Comment:
```suggestion
```

Reply:
This is to make it align cleanly with the tempest callout ending, should I still remove?

Reply:
Ahh okay.  In that case, it makes sense to me to override it to explicitly be 5 (as you did).